src/dotnet/commands/dotnet-sln/LocalizableStrings.cs

```diff
@@ -12,7 +12,7 @@ internal class LocalizableStrings
         public const string AppHelpText = "Projects to add or to remove from the solution.";
 
         public const string AddAppFullName = ".NET Add project(s) to a solution file Command";
-        public const string AddSubcommandHelpText = "Add a specified project(s) to the solution.";
+        public const string AddSubcommandHelpText = "Add one or more specified projects to the solution.";
```

Comment:
Add one or more specified projects is a bit awkward. It should be "Add the specified projects to the solution" or "Add one or more projects to the solution". From what I can I see in the SDK today, the latter would be fine.